src/ocamlorg_frontend/pages/releases.eml

```diff
@@ -13,69 +13,74 @@ Layout.render
-                                <a href="<%s Url.release release.version %>" class="text-primary-600 font-medium block">
+                            </div>
+                            <% releases |> List.iter (fun (release : Data.Release.t) -> %>
+                                <div class="flex flex-col lg:flex-row md:flex-col">
+                                    <div class="py-4 px-6 font-semibold lg:w-2/3 md:w-full w-full">
+                                        <div class="font-semibold">
+                                            <%s release.version %>
+                                        </div>
+                                        <div class="prose prose-orange font-normal">
+                                            <%s! release.highlights_html %>
+                                        </div>
+                                    </div>
+                                    <div class="lg:w-1/3 flex md:w-full w-full lg:flex-row md:flex-row md:justify-between flex-col">
+                                        <div class="py-4 px-6 font-medium align-top lg:w-1/2 md:w-auto w-full">
+                                            <%s release.date %>
+                                        </div>
+                                        <div class="py-4 px-6 align-top lg:w-1/2 md:w-auto w-full">
+                                            <a href="<%s Url.release release.version %>" class="text-primary-600 font-medium block">
                                     Release Notes
                                 </a>
-                                <a href="<%s Url.Package.overview "ocaml" ~version:release.version %>" class="text-primary-600 font-medium block">
+                                            <a href="<%s Url.Package.overview " ocaml " ~version:release.version %>" class="text-primary-600 font-medium block">
                                     OCaml Package
                                 </a>
-                                <a href="<%s Url.manual_with_version release.version %>" class="text-primary-600 font-medium block">
+                                            <a href="<%s Url.manual_with_version release.version %>" class="text-primary-600 font-medium block">
                                     Language Manual
                                 </a>
-                                <a href="<%s Url.api_with_version release.version %>" class="text-primary-600 font-medium block">
+                                            <a href="<%s Url.api_with_version release.version %>" class="text-primary-600 font-medium block">
                                     Standard Library API
                                 </a>
-                            </td>
-                        </tr>
+                                        </div>
+                                    </div>
+                                </div>
+                                <% ); %>
+                        </div>
```

Comment:
check that indentation matches the corresponding opening tag. I haven't checked, but looks suspicious.

Reply:
@sabine, The close tag corresponds to the code block used to check if there are releases to display and shows different templates based on whether releases exist or not. The indentation has been checked and is correctly aligned with the corresponding tags.